ansible/group_vars/sonic/sku-sensors-data.yml

```diff
@@ -491,9 +491,9 @@ sensors_checks:
       power: []
       temp:
       - - coretemp-isa-0000/Core 0/temp2_input
```

Comment:
seems that we are missing the CPU pack temp compare from day 1,  please refer to x86_64-mlnx_msn3700-r0,

      - - coretemp-isa-0000/\P[a-z]*\ id 0/temp1_input
        - coretemp-isa-0000/\P[a-z]*\ id 0/temp1_max
  
I would suggest adding it this time